maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/data/builder/TestModelDataBuilder.java

```diff
@@ -1125,8 +1134,8 @@ public static AssessmentStatusDTO getAssessmentStatusDTO(CurrentStatus status) {
 
     public static EmploymentStatusDTO getEmploymentStatusDTO() {
         return EmploymentStatusDTO.builder()
-                .code("EMPLOY")
-                .description("Employed")
+                .code(EMST_CODE)
+                .description(EMST_CODE)
```

Comment:
I'd prefer it if we weren't reusing the same value here.

Reply:
EMST_CODE is reusing other place and Changes mode for description